app/src/main/java/com/infomaniak/mail/utils/UiUtils.kt

```diff
@@ -130,4 +131,15 @@ object UiUtils {
     }
 
     fun dividerDrawable(context: Context) = AppCompatResources.getDrawable(context, R.drawable.divider)
+
+    fun formatSubBodiesContent(subBodies: List<SubBody>): String {
+        var subBodiesContent = ""
+        subBodies.forEach { subBody ->
+            subBody.bodyValue?.let {
+                if (subBodiesContent.isNotEmpty()) subBodiesContent += "<br/>"
+                subBodiesContent += "<blockquote>${it}</blockquote>"
+            }
+        }
```

Comment:
We don't need to somehow encapsulate child subbodies inside their parent?

Reply:
Nope, we do the simpler solution for this one.